joblib/test/test_parallel.py

```diff
@@ -573,16 +550,10 @@ def test_nested_exception_dispatch(backend):
     assert 'nested_function_inner' in report
     assert 'exception_raiser' in report
 
-    if PY3_OR_LATER:
-        # Under Python 3, there is no need for exception wrapping as the
-        # exception raised in a worker process is transportable by default and
-        # preserves the necessary information via the `__cause__` attribute.
-        assert type(excinfo.value) is ValueError
-    else:
-        # The wrapping mechanism used to make exception of Python2.7
-        # transportable does not create a JoblibJoblibJoblibValueError
-        # despite the 3 nested parallel calls.
-        assert type(excinfo.value) is JoblibValueError
+    # Under Python 3, there is no need for exception wrapping as the
+    # exception raised in a worker process is transportable by default and
+    # preserves the necessary information via the `__cause__` attribute.
```

Comment:
We can remove this comment. Instead we could improve this test (or another one to check that the `__cause__` attribute actually gives additional information.